internal/ignition/ignition.go

```diff
@@ -140,6 +141,23 @@ func (g *installerGenerator) Generate(ctx context.Context, installConfig []byte)
 		}
 	}
 
+	if *g.cluster.Operators.Lso.Enabled {
```

Comment:
`Operators` and `Lso` is nullable so i'm not sure that you can access `Enabled` directly without checking that the other parts are not `nil`

Reply:
I think that in RegisterCluster default values are set, so that should be OK.